rs/nervous_system/tools/sync-with-released-nervous-system-wasms/src/main.rs

```diff
@@ -173,9 +306,17 @@ struct CanisterUpdate {
     new_sha256: String,
 }
 
+#[derive(Clone, Debug)]
+struct ExternalCanisterUpdate {
```

Comment:
Nit. 

Perhaps I would slightly prefer something«to have a common type for the two cases, e.g.:

```
#[derive(Clone, Debug)]
struct CanisterUpdate {
    canister_name: String,
    git_hash: String,
    rev: Either<[u8; 64], String>,
}
```